src/main/java/org/jabref/gui/preferences/keybindings/KeyBindingsTab.java

```diff
@@ -71,6 +80,19 @@ private void initialize() {
                 .install(clearColumn);
 
         viewModel.keyBindingPresets().forEach(preset -> presetsButton.getItems().add(createMenuItem(preset)));
+
+        searchBox.textProperty().addListener((observable, previousText, searchTerm) ->
+                viewModel.filterValues(searchTerm));
+
+        ObjectProperty<Color> flashingColor = new SimpleObjectProperty<>(Color.TRANSPARENT);
+        StringProperty flashingColorStringProperty = ColorUtil.createFlashingColorStringProperty(flashingColor);
+
+        searchBox.styleProperty().bind(
+                new SimpleStringProperty("-fx-control-inner-background: ").concat(flashingColorStringProperty).concat(";")
+        );
+
+        searchBox.setPromptText(Localization.lang("Search") + "...");
```

Comment:
Always localize full sentences, including the ellipsis. Think about RTL languages. This would look very weird.

Reply:
This indeed comes from the present code:
https://github.com/JabRef/jabref/blob/7d480508632b4619cb662d4b9f5a1078990da827/src/main/java/org/jabref/gui/preferences/journals/JournalAbbreviationsTab.java#L88

Reply:
Existing code is not necessarily right. Always a lot to improve.
See https://stackoverflow.com/questions/48718469/how-is-ellipsis-localized

Reply:
so maybe i should adaadaptpt the current localization for search?

Reply:
To be fair, there are a lot of places where the ellipsis was used incorrectly under this circumstance, i.e.
<img width="495" alt="image" src="https://github.com/user-attachments/assets/031a09b7-7002-4064-aca2-03372dbf54ec">

I think the main focus of this PR is to provide the search feature to filter key binds, so I will go refactor only the ellipsis in the search localizaiton

Reply:
new code has been pushed in 71b10ca, and has been tested
<img width="329" alt="image" src="https://github.com/user-attachments/assets/c5d09421-a241-4029-9fa4-48a63e942786">
<img width="245" alt="image" src="https://github.com/user-attachments/assets/5e10b955-d16c-4664-a917-5b7dd08964f8">